src/code/z_collision_check.c

```diff
@@ -2169,6 +2174,8 @@ void CollisionCheck_ATTrisVsACCyl(PlayState* play, CollisionCheckContext* colChk
             }
 
             if (Math3D_CylTriVsIntersect(&acCyl->dim, &atTrisElem->dim, &hitPos) == true) {
+                Vec3f atPos;
+                Vec3f acPos;
                 atPos.x = (atTrisElem->dim.vtx[0].x + atTrisElem->dim.vtx[1].x + atTrisElem->dim.vtx[2].x) * (1.0f / 3);
```

Comment:
pretty small thing, but a style pattern we have kept pretty consistent for a long time is to always put a line break after variable declarations. its relevant in a few other places in this pr

Reply:
Good to know for the future, thanks